TickSpec.XunitWiring/FeatureSetup.fs

```diff
@@ -0,0 +1,66 @@
+﻿namespace TickSpec.XunitWiring
+
+open System.Collections.Generic
```

Comment:
nit: In general I've sorted these alphabetically rather than topologically throughout this repo

Reply:
fixed